core/src/main/resources/templates/clusters.html

```diff
@@ -517,13 +517,13 @@ <h4 class="mb-3">Shortcuts</h4>
 
 										<td> {{ allenv.protocol}} </td>
 
-										<td ng-show="allenv.clusterType == 'kafka'" >
+										<td ng-show="allenv.clusterType == 'KAFKA'" >
 											<span class="badge badge-success">{{ allenv.clusterType }}</span></td>
```

Comment:
In the ClusterFlavor column can we show a more user friendly version instead of the enum?

Reply:
Just as an FYI 
![image](https://github.com/Aiven-Open/klaw/assets/121855584/64c002a2-9131-4ac1-8ed6-b53c14a2b42b)

Reply:
Yeah, that's a problem that is introduced byt the enum change, but shouod probably be handled in a subsequent PR

Reply:
I'm also having a weird issue trying to connect to kafka to test the cluster creation. but that might be my local problem

Reply:
https://github.com/Aiven-Open/klaw/issues/2308